app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/partner/PartnerContactNameController.scala

```diff
@@ -49,64 +52,108 @@ class PartnerContactNameController @Inject() (
   def displayPage(): Action[AnyContent] =
```

Comment:
Perhaps rename to `displayNewPartner()` for consistency with the amend equivalent?